src/frontend/mame/clifront.cpp

```diff
@@ -568,27 +568,27 @@ void cli_frontend::listroms(const char *gamename)
 			osd_printf_info("\n");
 		first = false;
 		osd_printf_info("ROMs required for driver \"%s\".\n"
-				"Name                    Size Checksum\n", drivlist.driver().name);
+				"Name                                   Size Checksum\n", drivlist.driver().name);
 
 		// iterate through roms
 		for (device_t &device : device_iterator(drivlist.config().root_device()))
 			for (const rom_entry *region = rom_first_region(device); region; region = rom_next_region(region))
 				for (const rom_entry *rom = rom_first_file(region); rom; rom = rom_next_file(rom))
 				{
 					// accumulate the total length of all chunks
-					int length = -1;
+					uint32_t length = 0;
 					if (ROMREGION_ISROMDATA(region))
 						length = rom_file_size(rom);
 
 					// start with the name
 					const char *name = ROM_GETNAME(rom);
-					osd_printf_info("%-20s ", name);
+					osd_printf_info("%-32s ", name);
 
 					// output the length next
-					if (length >= 0)
-						osd_printf_info("%7d", length);
+					if (length >= 1)
+						osd_printf_info("%10u", length);
 					else
-						osd_printf_info("       ");
+						osd_printf_info("          ");
```

Comment:
Please change this to the form (once again to ease the job for the next person):
```
    osd_printf_info("%10s", "");
```